src/Microsoft.IdentityModel.Validators/AadTokenValidationParametersExtension.cs

```diff
@@ -17,6 +17,33 @@ namespace Microsoft.IdentityModel.Validators
     /// </summary>
     public static class AadTokenValidationParametersExtension
     {
+        /// <summary>
+        /// Enables validation of the cloud instance name of the Microsoft Entra ID token signing keys.
+        /// </summary>
+        /// <param name="tokenValidationParameters">The <see cref="TokenValidationParameters"/> that are used to validate the token.</param>
+        public static void EnableEntraIdSigningKeyCloudInstanceNameValidation(this TokenValidationParameters tokenValidationParameters)
```

Comment:
Perhaps shorten the name to: EnableEntraIdCoudInstanceValidation OR EnableEntraCloudInstanceValidation.
Not sure we need to be specific about Id or specify name as may expand critera.

Reply:
renamed to EnableEntraIdCoudInstanceValidation